alodataset/__init__.py

```diff
@@ -3,6 +3,7 @@
 from .split_mixin import SplitMixin
 from .waymo_dataset import WaymoDataset
 from .coco_detection_dataset import CocoDetectionDataset
+from .coco_segmentation_dataset import CocoSegementationDataset
```

Comment:
Doest CocoPanopticDataset would be more accurate ?

Reply:
OK